iteratee/src/main/scala/io/finch/iteratee/package.scala

```diff
@@ -0,0 +1,93 @@
+package io.finch
+
+import com.twitter.finagle.http.Response
+import com.twitter.io._
+import com.twitter.util.Future
+import io.catbird.util._
+import io.finch.internal._
+import io.finch.items.RequestItem
+import io.iteratee.{Enumerator, Iteratee}
+import io.iteratee.twitter.FutureModule
+import shapeless.Witness
+
+/**
+  * Iteratee module
+  */
+package object iteratee extends IterateeInstances {
+
+  import syntax._
+
+  private[finch] def enumeratorFromReader(reader: Reader): Enumerator[Future, Buf] = {
+    reader.read(Int.MaxValue).intoEnumerator.flatMap {

```

Comment:
Not here, but after function call. There is no sense to add `.ensure` for every chunk